compiler/test-resources/repl/i17231

```diff
@@ -0,0 +1,2 @@
+scala> summon[ValueOf["a"]]
+val res0: ValueOf["a"] = scala.ValueOf@61
```

Comment:
This `toString` uses the hash. This test will not be stable.

Reply:
Oh, I looked at the neighboring `i15493` which also had hashes, so I assumed it would be fine

Reply:
How do you advise I proceed ?

Reply:
The hash of ValueOf is of its underlying type, String, which is of the content.  "a" is hex 0x61, which isn't going to change.  So this is fine.